codegen/cmd/injection-gen/generators/namer.go

```diff
@@ -0,0 +1,58 @@
+/*
+Copyright 2016 The Kubernetes Authors.
```

Comment:
Don't want to change this boilerplate cause I copied it from `k8s.io/code-generator` repo